package.json

```diff
@@ -63,9 +64,11 @@
     "karma-mocha": "^0.2.2",
     "karma-webpack": "^1.7.0",
     "mocha-circleci-reporter": "0.0.1",
+    "node-static": "^0.7.7",
     "npm": "^3.8.8",
     "rimraf": "^2.5.2",
     "selenium-webdriver": "^2.53.2",
-    "style-loader": "^0.13.1"
+    "style-loader": "^0.13.1",
+    "todomvc": "^0.1.1"
```

Comment:
We're pulling it in so that the tests in CI can visit todomvc. I can replace this with a set of static built files, pretty easily. At some point, we might not want to test todomvc, but it seems like the easiest set of tests to write at the moment to touch on many different interactions (going between pages, breakpoints etc)... Also, there is some merit to having your test subject be more realistic.